src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -396,21 +444,9 @@ describe('ApplyForBenefits component', () => {
             initialState,
             reducers,
           });
-          expect(
-            view.getByRole('link', {
-              name: /apply for VA health care/i,
-            }),
-          ).to.exist;
-          expect(
-            view.getByRole('link', {
-              name: /learn how to file a claim/i,
-            }),
-          ).to.exist;
-          expect(
-            view.getByRole('link', {
-              name: /learn how to apply for education benefits/i,
-            }),
-          ).to.exist;
+          healthCareInfoIsShown(view);
+          claimsInfoIsShown(view);
+          educationInfoIsShown(view);
```

Comment:
So much nicer. Plus, now we could make these helper function more robust (check for more elements to be visible or hidden) very easily.